src/components/Theme/defaultTheme.ts

```diff
@@ -1,43 +1,81 @@
+const GRADIENT_DEGREE = {
+  '45': {useAngle: true, angle: 45, angleCenter: {x: 0.5, y: 0.5}},
+};
 export const defaultTheme = {
   //Will apply default font for ios and android(San Francisco on iOS and Roboto on Android)
   //Change this font in your project to apply changes on all components
   fontFamily: 'System',
+  primary: {
+    main: '#2400ff',
+    light: '#D1D3FD',
+    dark: '#17019C',
+    contrast: '#000000',
+  },
+  error: {
+    main: '#CC2418',
+    light: '#FBDCDA',
+    dark: '#B91D12',
+    contrast: '#000000',
+  },
+  success: {
+    main: '#088940',
+    light: '#B3DFCA',
+    dark: '#007133',
+    contrast: '#FFFFFF',
+  },
+  warning: {
+    main: '#FE9E34',
+    light: '#FFDDB3',
+    dark: '#EB621C',
+    contrast: '#000000',
+  },
+  grey: {
+    main: '#C7CCD3',
+    light: '#e4e7ec',
+    extraLight: '#EFF1F3',
+    dark: '#667985',
+  },
+  text: {
+    primary: '#000000',
+    secondary: '#344054',
+    disabled: '#D2D2D2',
+    hint: '#667085',
+  },
   background: {
-    main: '#FFF',
+    white: '#ffffff',
```

Comment:
Please, keep `main` color instead of `white` - names should not be related to specific color, since it can be changed